components/org.wso2.carbon.identity.media.core/src/test/java/org/wso2/carbon/identity/media/core/StorageSystemManagerTest.java

```diff
@@ -72,58 +73,6 @@ public void setUp() {
                         databaseBasedStorageSystemFactory);
     }
 
-    @Test
```

Comment:
why were these removed?

Reply:
these tests were removed as the media storage type will no longer be read from the identity.xml